Makefile

```diff
@@ -150,12 +150,15 @@ clean-qpid-testsuite:
 	$(gen_verbose) rm -rf qpid_testsuite
 
 prepare: test-dist create_ssl_certs
+	$(verbose) rm -rf plugins/rabbitmq_metronome*
```

Comment:
I like this solution. Because initially I decided to try to fix `metronome` instead )